homeassistant/components/generic/config_flow.py

```diff
@@ -261,6 +270,19 @@ async def async_test_stream(
     return {}
 
 
+def register_preview(hass: HomeAssistant):
+    """Set up previews for camera feeds during config flow."""
+    if not hass.data.get(DOMAIN):
+        hass.data[DOMAIN] = {}
+
+    if not hass.data[DOMAIN].get(PREVIEW):
+        _LOGGER.debug("Registering camera image preview handler")
+        preview = CameraImagePreview(hass)
+
+        hass.http.register_view(preview)
```

Comment:
let's go ahead and just inline this

```suggestion
        hass.http.register_view(CameraImagePreview(hass))
```